US-CBP/cbp-theme
# Create Application Header

#### What's this PR do?
this is the application header component.

#### Where should the reviewer start?
`applicationHeader.html`
`_cbp-appHeader.scss`

#### How should this be manually tested?
check to see if `applicationHeader.html` is displaying properly and the dropdown is functional in both desktop and mobile views.

#### Any background context you want to provide?
the dark background overlay and animation for when mobile nav is opened in tablet view still needs to be implemented.

#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
